components/sidebar/sidebar_category_list/sidebar_category_list.tsx

```diff
@@ -2,32 +2,286 @@
 // See LICENSE.txt for license information.
 
 import React from 'react';
+import Scrollbars from 'react-custom-scrollbars';
+import {FormattedMessage} from 'react-intl';
+
+import {Spring, SpringSystem, util as MathUtil} from 'rebound';
```

Comment:
In the HTML I'm building, I'm avoiding custom scrollbars. @deanwhillier thought this was something we should avoid. Does Spring just handle scroll position as opposed to overriding the default browser scrolling element?

Reply:
Ah, it looks like Spring is just the animation library https://www.react-spring.io/docs/hooks/basics

Reply:
Yep, that's right. It's just for having more control over the scrolling animation. We could look at removing it if it's not necessary, but it seems nice to have for now at least.